include/openssl/core_numbers.h

```diff
@@ -528,6 +538,8 @@ OSSL_CORE_MAKE_FUNC(int, OP_asym_cipher_decrypt, (void *ctx, unsigned char *out,
                                                   size_t inlen))
 OSSL_CORE_MAKE_FUNC(void, OP_asym_cipher_freectx, (void *ctx))
 OSSL_CORE_MAKE_FUNC(void *, OP_asym_cipher_dupctx, (void *ctx))
+OSSL_CORE_MAKE_FUNC(int, OP_asym_cipher_get_params, (const OSSL_PARAM params[]))
```

Comment:
Same here, I notice that in the documentation for aym_ciphers the `const` is correctly omitted.

Reply:
Fixed